### ISSUE TITLE
Docs: make "repository structure" easier to find

### ISSUE THREAD

**Issue by severo** [huggingface/datasets#5971]
The page https://huggingface.co/docs/datasets/repository_structure explains how to create a simple repository structure without a dataset script.
It's the simplest way to create a dataset and should be easier to find, particularly on the docs' first pages.

**Comment by mariosasko** [comment]
Loading a local dataset also works the same way when `data_files` are not specified, so I agree we should make this info  easier to discover 

cc @stevhliu

**Comment by benjaminbrown038** [comment]
self-assign

**Comment by mariosasko** [comment]
@benjaminbrown038 Yes, it is. Maybe @stevhliu can give some pointers on improving this doc page's discoverability.

**Comment by stevhliu** [comment]
I think we can add a version of the [Main use-case](https://huggingface.co/docs/datasets/repository_structure#main-usecase) section to the [Share a dataset to the Hub](https://huggingface.co/docs/datasets/upload_dataset) tutorial. 

Currently, it doesn't tell you *how* to structure the repository; it only tells you how to create it. So adding the "main use-case" will help bridge the gap and make it easier to find. We should also add a link to the [Structure your repository](https://huggingface.co/docs/datasets/repository_structure) guide for users who want to learn about the other options.

**Comment by benjaminbrown038** [comment]
#self-assign

**Comment by ShamikOfficial** [comment]
Hi! Opening a PR for this issue.

- Moved **Structure your repository** into the Tutorials section
- Linked it from the docs index and quickstart

#self-assign
